public/index.html

```diff
@@ -3,19 +3,18 @@
     <head>
         <meta charset="utf-8">
         <title>Chat</title>
+        <!-- external libraries !-->
         <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular.min.js"></script>
```

Comment:
Okay, would that be purely to improve readability of the HTML page? Would there be any other advantages considering the external libraries are already optimized/minified?